scripts/README.md

```diff
@@ -5,9 +5,10 @@ This package contains shell scripts and libraries used for running e2e integrati
 ### run-test.sh
 
 `run-test.sh` can run various integration test suites against the current revision in the invoking directory. 
```

Comment:
Since this is used only for GH actions, we can add a note here..make targets can be added as a follow up..